### PR TITLE
Nuget package + release notes

### PR DESCRIPTION
**Pull request by KevinRansom**
Transfer FCS release notes into release-notes.md.

Add new sections to the release notes:
````
### FSharpLanguage 5.0.0
* Fix () - Some bug fix [A Developer](https://github.com/Adeveloper).

### FSharp Core 5.0.0
* Fix () - Some bug fix [A Developer](https://github.com/Adeveloper).

### FSharp Tools 11.0.0
* Fix () - Some bug fix [A Developer](https://github.com/Adeveloper).

### FSharp Compiler Service 37.0.0
* Fix () - Some bug fix [A Developer](https://github.com/Adeveloper).
````

The release notes will be maintained in the main branch of the dotnet/fsharp repo.  The nuget packages wll contain a link to them.

This pr: also generate uri's for release notes element of nuspec file that point to the relevent sections.

FSharp.Core uri looks similar to:
    https://github.com/dotnet/fsharp/blob/main/release-notes.md#FSharp-Core-5-0-0

Compiler.Service:
    https://github.com/dotnet/fsharp/blob/main/release-notes.md#FSharp-Compilere-Service-37-0-0

### CODE REVIEW

**Comment by baronfel** [comment]
I love it. Quick note on markdown: if you don't want to have to duplicate attribution for folks, you can use [reference-style-links](https://www.markdownguide.org/basic-syntax/#reference-style-links) to only have to define the link's address once, and reuse it in all other locations.

File: src/fsharp/FSharp.Core/FSharp.Core.fsproj [hunk 1]
```diff
@@ -17,7 +17,8 @@
     <PackageId>FSharp.Core</PackageId>
     <NuspecFile>FSharp.Core.nuspec</NuspecFile>
     <IsPackable>true</IsPackable>
-    <PackageDescription>FSharp.Core redistributables from Visual F# Tools version $(FSPackageMajorVersion) For F# $(FSCoreMajorVersion).  Contains code from the F# Software Foundation.</PackageDescription>
+    <PackageDescription>FSharp.Core redistributables from Visual F# Tools version $(FSCorePackageVersion) For F# $(FSLanguageVersion).  Contains code from the F# Software Foundation.</PackageDescription>
```
Comment (cartermp) [inline src/fsharp/FSharp.Core/FSharp.Core.fsproj:20]:
```suggestion
    <PackageDescription>FSharp.Core redistributables from F# Tools version $(FSCorePackageVersion) For F# $(FSLanguageVersion).  Contains code from the F# Software Foundation.</PackageDescription>
```

Let's finally drop the "Visual" from here :)

Comment (KevinRansom) [reply src/fsharp/FSharp.Core/FSharp.Core.fsproj:20]: The fcs release notes are already in, they are at the bottom of the release-notes file.
Yes I can yeet the Visual from Visual FSharp.

Comment (cartermp) [reply src/fsharp/FSharp.Core/FSharp.Core.fsproj:20]: Yee

File: eng/Versions.props [hunk 2]
```diff
@@ -11,29 +11,46 @@
   </PropertyGroup>
   <!-- Version number computation -->
   <PropertyGroup>
+
```
Comment (brettfo) [inline eng/Versions.props:14]: Nit: Arcade updates will remove the newlines in this file.

**Pull request merged**